src/Components/LandingPage/ImagesEmpty.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+import { Title } from '@patternfly/react-core';
+import { Table, Tbody, Td, Th, Thead, Tr } from '@patternfly/react-table';
+
+const ImagesEmpty = () => (
```

Comment:
This isn't really adding anything, I think it would be best to just leave it out. The real challenge is reusing existing components, and as such, it's unlikely this will ever be used.